### PR TITLE
Nukes true/false for TRUE/FALSE

### PR DESCRIPTION
**Pull request by Zandario**
[INTERNAL][CODE QUALITY]
## About the PR 
I got reminded that true/false existed on Goon and while doing my normal work I did this.

## Why's this needed? 
Having two identical sets of preprocessor TRUE/FALSE macros is kinda pointless.

### CODE REVIEW

File: code/modules/medical/medicaldiagnosis.dm [hunk 1]
```diff
@@ -45,7 +45,7 @@
 						if(!user.traitHolder.hasTrait("training_medical") && prob(15))
 							boutput(user, "<span class='alert'>You attempt to listen to [(user != H) ? "[H]'s" : "your"] lungs before realizing after a few attempts that you've been listening to [(user != H) ? "[H]'s" : "your"] [pick("liver", "kidneys", "spleen", "leg", "PDA", "eyes")], a shameful [user]</span>")
 						else
-							if(isalive(H) == false)
+							if(isalive(H) == FALSE)
```
Comment (frawhst) [inline code/modules/medical/medicaldiagnosis.dm:48]:
```suggestion
							if(!isalive(H))
```

File: code/obj/item/stun_baton.dm [hunk 2]
```diff
@@ -46,7 +46,8 @@ TYPEINFO(/obj/item/baton)
 	var/can_swap_cell = 1
 	var/rechargable = 1
 	var/beepsky_held_this = 0 // Did a certain validhunter hold this?
-	var/flipped = false //is it currently rotated so that youre grabbing it by the head?
+	/// Is it currently rotated so that youre grabbing it by the head?
```
Comment (frawhst) [inline code/obj/item/stun_baton.dm:49]:
```suggestion
	/// Is it currently rotated so that you're grabbing it by the head?
```

**Review by frawhst — APPROVED** [review]
skimmed, lgtm other than a couple tiny things

**Review by ZeWaka — APPROVED** [review]

**Comment by pali6** [comment]
Make sure you do changes like this correctly :/ This ignores all usage of the true/false macros in #ifdefs and as such it broke RP server builds for half a day

**Pull request merged**